src/app/core/pipes/personal-achievements-date.pipe.ts

```diff
@@ -0,0 +1,25 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'personalAchievementsDate'
+})
+export class PersonalAchievementsDatePipe implements PipeTransform {
+
+  transform(value: Date): string {
+
+  debugger
+
```

Comment:
Remove these lines. And always check to remove all console.log and debuggers before you commit any changes.

Reply:
fixed